setup.py

```diff
@@ -5,6 +5,8 @@
 
 from setuptools import setup
 
+sys.path.insert(0, ".")
```

Comment:
Why we need this?

Reply:
The first workflow failed with error on import django so I included it in path

Reply:
Editable install should just work according the comments in ticket-33778 :thinking: (\cc @mgorny)

Reply:
mI agree, I will remove the line and see if workflow succeed

Reply:
I agree, I don't see why `sys.path` manipulation would be needed there.

Reply:
@mgorny, why do u see sys.path not needed ??

Reply:
Because I don't see any `import` statements below it.